entity.go

```diff
@@ -82,3 +82,24 @@ func (e *Entity) ComponentFast(c Component) interface{} {
 func (e *Entity) ID() string {
 	return e.id
 }
+
+// Duplicate populates an array with the specified amounts of clones from an Entity.
+// NOTE: If you provide your components as pointers, the pointer will be duplicated.
+// Hence, the entities will "share" a component.
```

Comment:
I know we can add them as non-pointers, but unless you get a pointer to it by using `entity.Component()`, you can't actually change the data.